src/openrct2/rct2/S6Importer.cpp

```diff
@@ -252,7 +252,15 @@ namespace RCT2
 
             gInitialCash = ToMoney64(_s6.initial_cash);
             gBankLoan = ToMoney64(_s6.current_loan);
-            gParkFlags = _s6.park_flags;
+
+            gParkFlags = _s6.park_flags & ~PARK_FLAGS_NO_MONEY_SCENARIO;
+
+            // RCT2 used a different flag for `no money` when the park is a scenario
+            if (_s6.header.type == S6_TYPE_SCENARIO && (_s6.park_flags & PARK_FLAGS_NO_MONEY_SCENARIO))
```

Comment:
This contains an error. If the save if not a scenario, but a regular save, this will _always_ turn the `PARK_FLAGS_NO_MONEY` flag off.

Reply:
Looks like only the `else` block has to be removed in that case.

Reply:
Just tested it, and I can confirm that with the else-block removed, the game loads .SV6 parks that were saved with no money correctly.